test/shared/src/main/scala/zio/test/PrettyPrint.scala

```diff
@@ -1,6 +1,6 @@
 package zio.test
 
-import zio.internal.ansi.AnsiStringOps
+import zio.internal.ansi.AnsiStringOps._
```

Comment:
Stray change.

Reply:
done and also changed renderAssertionResult to always return a Message